sql/2021/resource-hints/consoleLog_missing_crossorigin.sql

```diff
@@ -0,0 +1,45 @@
+#standardSQL
+# returns the file types for preload tags without the required crossorigin attribute
+
+# helper to create percent fields
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+# helper to convert the href into a type
+CREATE TEMP FUNCTION GET_TYPE (href STRING) RETURNS STRING AS (
+  IF(
+    REGEXP_CONTAINS(href, r'fonts\.googleapis\.com'),
+    "fonts.googleapis.com",
+    TRIM(TRIM(REGEXP_EXTRACT(href, r'\.[0-9a-z]+(?:[\?#]|$)'), '?'), '#')
```

Comment:
The function sounds like it's returning a "type" but the value seems more like a host name or path name. Do any of the [`NET`](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#net_functions) functions do what you're looking for (HOST, REG_DOMAIN)?